src/requests/cache/mod.rs

```diff
@@ -7,6 +7,7 @@ pub mod flush_cache;
 pub mod list_caches;
 
 pub mod basic;
+pub mod scalar;
```

Comment:
nit - I put the `Get` request and friends in `basic` and I see you're adding `scalar` here. I don't have a strong pref which of those we go with but I'd probably just stick with one or the other. It's kinda the "everything except collections" directory IMO. but if you think they should be separated lmk!

Reply:
I think "scalar" is more in line with the other SDKs. I can come back to clean this up when doing the cleaning-up-exports ticket